alueError: Invalid key set format
156.17.235.61 - - [24/Oct/2023 17:03:59] "GET /login/authorized?state=2jJmuZVo7w2KEYIQgTGxdAbYMsZD5S&code=4/0AfJohXmYfZ2lPdyFoMubtYHySTzNVpqs27JSV39aFtHK1qC9Kwb2CllFHy7aqo4DxQMwkQ&scope=email+profile+https://www.googleapis.com/auth/userinfo.email+openid+https://www.googleapis.com/auth/userinfo.profile&authuser=1&hd=student.pwr.edu.pl&prompt=none HTTP/1.1" 500 -
156.17.235.61 - - [24/Oct/2023 17:11:56] "GET / HTTP/1.1" 200 -
156.17.235.61 - - [24/Oct/2023 17:11:57] "GET /static/css/main.css HTTP/1.1" 304 -
156.17.235.61 - - [24/Oct/2023 17:12:00] "GET /login HTTP/1.1" 302 -
[2023-10-24 17:12:04,598] ERROR in app: Exception on /login/authorized [GET]
Traceback (most recent call last):
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/259441/cook-smarter/app.py", line 43, in authorized
    token = google.authorize_access_token()
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/authlib/integrations/flask_client/apps.py", line 105, in authorize_access_token
    userinfo = self.parse_id_token(token, nonce=state_data['nonce'], claims_options=claims_options)
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/authlib/integrations/base_client/sync_openid.py", line 66, in parse_id_token
    claims = _jwt.decode(
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/authlib/jose/rfc7519/jwt.py", line 96, in decode
    data = self._jws.deserialize_compact(s, load_key, decode_payload)
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/authlib/jose/rfc7515/jws.py", line 101, in deserialize_compact
    algorithm, key = self._prepare_algorithm_key(jws_header, payload, key)
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/authlib/jose/rfc7515/jws.py", line 254, in _prepare_algorithm_key
    key = key(header, payload)
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/authlib/integrations/base_client/sync_openid.py", line 38, in load_key
    jwk_set = JsonWebKey.import_key_set(self.fetch_jwk_set())
  File "/home/259441/cook-smarter/venv/lib/python3.8/site-packages/authlib/jose/rfc7517/jwk.py", line 55, in import_key_set
    raise ValueError('Invalid key set format')
ValueError: Invalid key set format
156.17.235.61 - - [24/Oct/2023 17:12:04] "GET /login/authorized?state=INEwCPYXtMMbcYByFtlG9y54ifBxFb&code=4/0AfJohXmuiVTriT0Ncz394Kp0xb3husLu9TBmH3IN71Q4O4sCTgSUOMPjd1WpyqoRGTtSww&scope=email+profile+https://www.googleapis.com/auth/userinfo.profile+openid+https://www.googleapis.com/auth/userinfo.email&authuser=3&prompt=none HTTP/1.1" 500 -
